src/conf/config.js

```diff
@@ -1,6 +1,7 @@
 
 const app_config = {
 //	FYC_API_URL: process.env.FYC_API_URL,
+//	FYC_API_URL: "https://api-dev.findyourchurch.org",
 	FYC_API_URL: "https://api-uat.findyourchurch.org",
```

Comment:
So this is not being injected by environment variables. I shouldn't see the URL in here.

Reply:
How about using a .env file? (FE can't access system environment variables.)
Our .env file can include:
`FYC_API_URL="https://api-uat.findyourchurch.org"`
To use this way, we must some node modules. I will notify you when I am absolutely sure about this.

Reply:
@serhiy1017 I'm sorry, but I thought I replied to this. Yes lets do a .env file.